manifests/base/kustomization.yaml

```diff
@@ -21,355 +21,644 @@ resources:
   - runtime-rocm-tensorflow-imagestream.yaml
   - runtime-tensorflow-imagestream.yaml
 
-commonLabels:
-  opendatahub.io/component: "true"
-  component.opendatahub.io/name: notebooks
 
 configMapGenerator:
-  - name: notebooks-parameters
-    env: params.env
-  - name: notebook
-    env: commit.env
+  - envs:
+      - params.env
+    name: notebooks-parameters
+  - envs:
+      - commit.env
+    name: notebook
 generatorOptions:
   disableNameSuffixHash: true
-vars:
-  - name: odh-minimal-notebook-image-n
-    objref:
+
+labels:
+  - includeSelectors: true
```

Comment:
got confused by docs whether we do or do not want these includeXYZ settings; will look again later